src/styles/sidebar/components/SelectionTabs.scss

```diff
@@ -33,6 +33,7 @@
   cursor: pointer;
   min-width: 85px;
   min-height: 18px;
+  border-radius: var.$border-radius;
```

Comment:
The background color is transparent, so this has no visual effect, but it does effect the :focus outline `box-shadow`, and makes it look slightly less squared. The native focus outline has a curved radius and so do our buttons so I applied the same radius to `<a>` tags which do require keyboard focus styling.